### PR TITLE
Add onClickCallback

### PR DESCRIPTION
**Pull request by JeremyEllingham**
I needed to be able to pass a function that triggers on clicking the PDF itself (and separate from the navigation controls), so here it is for anyone to be able to use.

### CODE REVIEW

File: dist/mgr-pdf-viewer-react.js [hunk 1]
```diff
@@ -436,6 +441,7 @@ PDFViewer.propTypes = {
   page: PropTypes.number,
   scale: PropTypes.number,
   css: PropTypes.string,
+  onClickCallback: PropTypes.any,
```
Comment (MGrin) [inline dist/mgr-pdf-viewer-react.js]: Can you change it for `PropTypes.func` please?

File: src/index.js [hunk 2]
```diff
@@ -120,6 +122,7 @@ PDFViewer.propTypes = {
   page: PropTypes.number,
   scale: PropTypes.number,
   css: PropTypes.string,
+  onClickCallback: PropTypes.any,
```
Comment (MGrin) [inline src/index.js]: same here :)

File: dist/mgr-pdf-viewer-react.js (hunk 1 above)
Comment (JeremyEllingham) [reply dist/mgr-pdf-viewer-react.js]: Ah yes, I should have checked the docs, I'm not familiar with PropTypes.

**Comment by JeremyEllingham** [comment]
Thank you for the excellent library, it's exactly what I was looking for (minus the onClickCallback!).  If there's any further improvements necessary in house I'll try generalise them for the library and put in another PR.

**Comment by MGrin** [comment]
@JeremyEllingham I changed the name of the prop to `onDocumentClick` - this name is more aligned with React coding standard namings

Will publish new version now

**Pull request merged**